enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -3319,6 +3319,14 @@ func (p *PebbleCache) Stop() error {
 	}
 	log.Infof("Pebble Cache [%s]: waitgroups finished", p.name)
 
+	// Flushed db after all waitgroups finished to reduce the change of lost
+	// evictions during app restarts
+	p.flushPartitionMetadata()
+	if err := p.db.Flush(); err != nil {
+		return err
+	}
+	log.Infof("Pebble Cache [%s]: db flushed", p.name)
```

Comment:
can you change this log message slightly so we can differentiate this log message from the other flush?